src/translations/en.json

```diff
@@ -1668,8 +1668,9 @@
                   "seconds": "Seconds"
                 },
                 "webhook": {
+                  "copy_url": "Copy URL to Clipboard",
                   "label": "Webhook",
-                  "webhook_id": "Webhook ID"
+                  "webhook_id": "Webhook ID (treat this like a password, use a unique secret value)"
```

Comment:
This should be set as a helper text, not appended to the label.

https://material.io/components/text-fields#anatomy

Reply:
I also think that the helper text can be a bit more user friendly.

> Anyone with this ID can trigger this webhook so make it hard to guess.

Reply:
"Anyone with this ID can trigger this webhook so make it hard to guess." -- not strictly true, you _also_ need access to connect to the instance of Home Assistant.

Perhaps (this is a bit long...):

> Webhook triggers are not authenticated. Anything on your network, including malicious users, devices, or software, can trigger the webhook if they know the ID. Treat it like a password: keep it secret, and make it hard to guess.

Since this would be helper text instead of the label, perhaps length isn't as much of an issue?

Reply:
I'm struggling a bit with the CSS on this one, but I think I almost have it.

I think the biggest concern is for those who expose HA directly to the internet. Any random web page can also trigger the webhook if you're browsing from the same network as HA.

Here are some mock-ups. I think I prefer the one-liners.

![image](https://user-images.githubusercontent.com/289218/152902645-ba7c2a32-4934-4798-8d67-6c71d14ab24d.png)

![image](https://user-images.githubusercontent.com/289218/152902523-911e7766-4d4c-4024-b2cd-a6bbac790ddf.png)

![image](https://user-images.githubusercontent.com/289218/152902335-81acc0dd-32db-46e3-8a5b-4d6427418af7.png)

Reply:
In ended-up going with "*Treat this ID like a password: keep it secret, and make it hard to guess.*" in [edf5645](https://github.com/home-assistant/frontend/pull/11568/commits/edf5645450d299147cd5f917775d15cd277e1cc6). The middle dialog above. Hope that's okay.

I feel like the helper text should be a little closer vertically to the paper-input. But I don't know how to make that happen. If you also feel the same, and know how to fix it, please suggest a fix.